pages/capitalization.md

```diff
@@ -15,6 +14,11 @@ capitalization guidelines:
 
 -   Don't capitalize “federal” or “government”
 
+When you're deciding whether to capitalize noun phrases, keep in mind that in English, title case is often a marker of formality. Using it judiciously can help clarify that you're speaking about a specific, official thing (such as a form, office, or person). Overuse can cause users stress by implying formality or officialness where it doesn't exist. For instance:
```

Comment:
Can we use “official status” here instead of “officialness”? (IMO “officialness” *is* a word but sounds odd… although I can see the argument for using it here precisely to underscore the point about not seeming overly serious.)